apps/include/ca.h

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Rename this as apps_ca.h?

Reply:
I chose the name because it is the header for `apps/ca.c`. To be consistent we could move stuff out of `ca.c` and create `lib/apps_ca.c`. As a first step we should agree how you prefer the separation, I'll keep this for now and keep it in mind...